sessions/react-global-state/demo-end/pages/components/MainContent.js

```diff
@@ -0,0 +1,60 @@
+import styled from "styled-components";
+import Counter from "./Counter";
+import Link from "next/link";
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: var(--accent-color);
+`;
+
+const List = styled.ul`
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  gap: 1rem;
+`;
+
+export default function MainContent({ animals, handleAdd, handleSubtract }) {
```

Comment:
I ask myself why this is called "main" content, an the the content on other pages not "main" enough.

Maybe `Counters` is more describing and less conflicting naming.

Reply:
I'd rather not name it counters, because it doesn't contain purely counters. Perhaps we could separate the component into CounterList and Quote and move them to the index page.